src/config_loader.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, file_name='config.ini'):
 
         self.max_response_sentences = int(config['LanguageModel']['max_response_sentences'])
         self.llm = config['LanguageModel']['model']
+        self.alternative_openai_api_base = config['LanguageModel']['alternative_openai_api_base']
```

Comment:
If we want this to not always be set maybe you should convert this to `None` if none, so you simply can use `s is None` instead of `s == 'none'`